integration/shareddata/composites.go

```diff
@@ -42,7 +43,43 @@ var Composites = &Values[string]{
 		"array-null":          bson.A{nil},
 		"array-numbers-asc":   bson.A{int32(42), int64(43), 45.5},
 		"array-strings-desc":  bson.A{"c", "b", "a"},
-		"array-composite":     bson.A{nil, primitive.Timestamp{T: 42, I: 13}, false, "z", primitive.NilObjectID},
+		"array-documents":     bson.A{bson.D{{"field", int32(42)}}, bson.D{{"field", int32(44)}}},
+		"array-composite": bson.A{
```

Comment:
As agreed in the call, maybe this document could be shorter and could contain only one value of each type + corner cases (so it's easier to maintain it in the future)